org.ektorp/src/main/java/org/ektorp/impl/StdObjectMapperFactory.java

```diff
@@ -47,7 +47,7 @@ public void setWriteDatesAsTimestamps(boolean b) {
 
 	private void applyDefaultConfiguration(ObjectMapper om) {
```

Comment:
I don't think I'd use that. I've just reimplemented the interface to respect the given `ObjectMapper` in my application. Like you mentioned above, I think it makes sense for the developer to define what custom configuration looks like. There is some configuration I'd like to be universal, though.

```java
final ObjectMapper mapper = new ObjectMapper();
mapper.registerModule(new CouchDbSerializationModule());
return new StdCouchDbConnector("sentry", new StdCouchDbInstance(client), new ObjectMapperFactory() {
      @Override public ObjectMapper createObjectMapper() {
          return mapper;
      }

      @Override public ObjectMapper createObjectMapper(CouchDbConnector connector) {
          mapper.registerModule(new EktorpJacksonModule(connector, mapper));
          return mapper;
      }
});
```